docs/src/modules/components/DemoSandboxed.js

```diff
@@ -10,10 +10,10 @@ import DemoErrorBoundary from 'docs/src/modules/components/DemoErrorBoundary';
 function FramedDemo(props) {
   const { children, document } = props;
 
-  const { direction } = useTheme();
+  const theme = useTheme();
   React.useEffect(() => {
-    document.body.dir = direction;
-  }, [document, direction]);
+    document.body.dir = theme.direction;
+  }, [document, theme]);
```

Comment:
Please revert. This will fire unnecessarily.

Reply:
Why is over firing an issue?

Reply:
Resolved with https://github.com/mui-org/material-ui/pull/21770

Reply:
Perfect :)